### PR TITLE
Updating README.md for contributors 

### PR DESCRIPTION
**Pull request by Anthonyklassen**
Updating README.md for visibility for new contributors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+  ---------
+  ii. Contributing
+  -----------------
+  iii. License
+  -----------------
+  iv. Installing
+  -----------------
+  v. More Information
+  -------------------
+
```
Comment (Pentarctagon) [inline README.md:15]: If you're going to add a table of contents, it should link to the entries it lists.

File: README.md [hunk 2]
```diff
@@ -8,6 +21,53 @@ high fantasy theme, featuring both singleplayer and online/hotseat multiplayer
 combat. Fight a desperate battle to reclaim the throne of Wesnoth, or take
 hand in any number of other adventures.
 
+Contributing
+-------------
+Interested in creating your content? One of Wesnoth's best features is its extensibility. Players can create new maps, units, races, scenarios, art, music, and even entire campaigns! Please read information on our site (https://wiki.wesnoth.org/Create) for more information.
```
Comment (Pentarctagon) [inline README.md:26]: > Interested in creating your content?

Should be "Interested in creating your **own** content?"

Comment (Pentarctagon) [inline README.md:26]: > Please read information on our site (https://wiki.wesnoth.org/Create) for more information.

Please change this to "To get started, see the [Create](https://wiki.wesnoth.org/Create) page on the wiki."

File: README.md [hunk 3]
```diff
@@ -8,6 +21,53 @@ high fantasy theme, featuring both singleplayer and online/hotseat multiplayer
...
+
+**What can players contribute?!**
```
Comment (Pentarctagon) [inline README.md:28]: Remove the exclamation point.

Comment (Pentarctagon) [reply README.md:28]: > What can players contribute?

should be "Here are some additional resources:"

File: README.md [hunk 4]
```diff
@@ -8,6 +21,53 @@ high fantasy theme, featuring both singleplayer and online/hotseat multiplayer
...
+(https://wiki.wesnoth.org/Wesnoth_AI)
+
+-Translations - work on translating Wesnoth
+(https://wiki.wesnoth.org/WesnothTranslations)
+
+-Authoring tools - tools for helping you write campaign WML
+(https://wiki.wesnoth.org/Authoring_tools)
+
+-Maintenance tools - tools for helping you sanity-check and maintain campaigns
+(https://wiki.wesnoth.org/Maintenance_tools)
```
Comment (Pentarctagon) [inline README.md:70]: All of this should link via the text, rather than having them separately in parenthesis.

File: README.md [hunk 5]
```diff
@@ -8,6 +21,53 @@ high fantasy theme, featuring both singleplayer and online/hotseat multiplayer
 combat. Fight a desperate battle to reclaim the throne of Wesnoth, or take
 hand in any number of other adventures.
 
+Contributing
+-------------
+Interested in creating your content? One of Wesnoth's best features is its extensibility. Players can create new maps, units, races, scenarios, art, music, and even entire campaigns! Please read information on our site (https://wiki.wesnoth.org/Create) for more information.
+
+**What can players contribute?!**
+
+ -WML Tutorial - Introduction to WML (In Progress)
```
Comment (Pentarctagon) [inline README.md:30]: Introduction -> introduction, for consistency with the rest of the list below it.

**Comment by soliton-** [comment]
A lot of this just duplicates the https://wiki.wesnoth.org/Create page. Instead of maintaining this in two places the link to Create should be enough IMO.

**Comment by cooljeanius** [comment]
> A lot of this just duplicates the https://wiki.wesnoth.org/Create page. Instead of maintaining this in two places the link to Create should be enough IMO.

Some people might not bother clicking thru, tho, so I think it's ok to duplicate a little bit of it here for people who only read READMEs.

**Pull request closed without merging**